src/build/search.js

```diff
@@ -0,0 +1,54 @@
+const path = require("path");
+const fs = require("fs").promises;
+const MiniSearch = require("minisearch");
+
+/* These words are too common to provide any real search value. Ignoring them
+ * improves search relevancy and reduces the index filesize. Use lowercase.
+ */
+const STOP_WORDS = {
+  en: new Set([
+    "halo", "and", "or", "not", "to", "from", "at", "in", "a", "the", "be", "are",
+    "is", "as", "its", "it", "this", "that", "these", "any", "e", "g", "for"
+  ]),
+  es: new Set([
+    "halo", "y", "o", "no", "a", "de", "en", "una", "uno", "la", "el", "ser",
+    "son", "es", "como", "sus", "está", "eso", "esta", "ese", "esa", "estas",
+    "estos", "alguna", "alguno", "p", "ej", "por"
+  ])
```

Comment:
Not much new about the search index except the addition of Spanish stop words and saving one index per language.